pike/crypto.py

```diff
@@ -162,7 +166,7 @@ def _encode_header(self, cur):
         # the crypto header will be written in _encode_smb2
         self.crypto_header_offset = cur.offset
         # save a space for the wire nonce
-        self.wire_nonce_hole = cur.hole.encode_bytes('\0'*16)
+        self.wire_nonce_hole = cur.hole.encode_bytes(b'\0'*16)
```

Comment:
Not sure if you care about PEP8 spacing for this PR.

Reply:
i'm not going super crazy cleaning everything up.

there's a lot of style stuff going on in pike, and i plan to address it all with black "soon"